lib/dal/src/func/authoring/save.rs

```diff
@@ -411,27 +441,25 @@ pub(crate) async fn save_attr_func_proto_arguments(
             AttributePrototypeArgument::remove_or_no_op(ctx, arg.id).await?;
         }
 
-        let attribute_prototype_argument =
```

Comment:
I don't believe this code is actually called right now, because this form of APA::new() already doesn't exist. Fixed the code anyway.